src/main.py

```
@@ -1,253 +1,206 @@
+    ip_min = int(getattr(env, "RL_IP_PER_MINUTE", "5"))
+    ip_hr = int(getattr(env, "RL_IP_PER_HOUR", "50"))
+    ip_day = int(getattr(env, "RL_IP_PER_DAY", "200"))
+    em_hr = int(getattr(env, "RL_EMAIL_PER_HOUR", "10"))
+    emd_hr = int(getattr(env, "RL_EMAIL_DOMAIN_PER_HOUR", "20"))
+    de_iphr = int(getattr(env, "RL_DISTINCT_EMAILS_PER_IP_PER_HOUR", "10"))
+    glob_min = int(getattr(env, "RL_GLOBAL_PER_MINUTE", "30"))
+
+    org_email_lc = org_email.lower()
+    email_domain = org_email_lc.split("@")[-1] if "@" in org_email_lc else "invalid"
+
+    if _limit_exceeded(_incr_counter(f"g:m:{b_min}"), glob_min): return "global rate limit exceeded (per minute)"
```

Ruff (0.15.5)

[error] Multiple statements on one line (colon)

(E701)